Tests/SWBCoreTests/SettingsTests.swift

```diff
@@ -1957,8 +1957,8 @@ import SWBMacro
         }
 
         let scope = settings.globalScope
-        #expect(scope.evaluate(BuiltinMacros.SYMROOT) == Path("/tmp/Workspace/aProject/build"))
-        #expect(scope.evaluate(BuiltinMacros.DSTROOT) == Path("/tmp/Workspace/aProject/build/foo/bar/dst"))
+        #expect(scope.evaluate(BuiltinMacros.SYMROOT) == Path.root.join("/tmp/Workspace/aProject/build"))
```

Comment:
Need to remove the leading slash from the call to join. Maybe it works by accident right now, but this is not semantically correct (on Windows you want a path relative to the left hand side, not relative to the current drive root). Same thing in BuildDescriptionTests.swift